src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public static final String REPORT_TITLE_PATTERN = "Report for period %s - %s";
+    public static final String PERSON_DESCRIPTION_PATTERN = "%s - %s";
+    public static final String DATA_SEPARATOR = " ";
+    public static final int DATE_POSITION = 0;
+    public static final int NAME_POSITION = 1;
+    public static final int HOURS_POSITION = 2;
+    public static final int INCOME_POSITION = 3;
+    private static final DateTimeFormatter D_FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
```suggestion
    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```
better use full words in names

Reply:
Got it